views/layouts/default.hbs

```diff
@@ -1,77 +1,36 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
+    
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>{{ title }}</title>
+    <link href="https://fonts.googleapis.com/css?family=Fira+Sans:300,400,500,600" rel="stylesheet">
```

Comment:
Does this need to be specified in the HTML _and_ in the CSS?

```css
@import url('https://fonts.googleapis.com/css?family=Fira+Sans:300,400,500,600');
```

&mdash; per https://github.com/mozilla/blurts-server/pull/123/files?short_path=e0d7738#diff-f2f4863f79a70d8706d0df15af6012d0R1

Reply:
So I'm wondering if the font import needs to be there at all. If the plan is still to test this initially on Windows only desktops, then I think (but need to revisit the Photon documentation) the font specified for Windows users is "Segoe UI" which is potentially a system font that doesn't need to be imported. So maybe we can do away with this entirely.

Reply:
Also, should we link out a 3rd party google site, or should we use the Fira CDN?
https://github.com/mozilla/Fira#usage
I don't have a lot of knowledge here to know if our https://code.cdn.mozilla.net/fonts/fira.css CDN is the same as the Google version and has all the same weights. Although I guess I could try loading both fonts and see what the network responses are...